sos/plugins/networking.py

```diff
@@ -207,7 +215,23 @@ def setup(self):
                         or line.isspace() \
                         or line[:1].isspace():
                     continue
-                out_ns.append(line.partition(' ')[0])
+                # if namespaces defined, remove from out_ns not matching
+                # namespaces with regex
+                if self.get_option("namespaces"):
+                    regex = '(?:%s)' % '|'.join(
+                        self.get_option("namespaces").split())
```

Comment:
Maybe worth moving this variable initialization ahead of `for line in ..` cycle, to set it once, and not every time you loop inside the cycle?

Reply:
regex initialization moved out from for loop